sql/_27_banana_qa/issue_5765_timezone_support/_01_table_operations/_02_alter/answers/_01_alter_add_hard_default_yes_tsltz.answer

```diff
@@ -131,7 +131,7 @@ a    c_tsltz1    c_tsltz2    c_tsltz3
 Error:-494
 ===================================================
```

Comment:
Shouldn't it be changed to the same content below?
```
sql/_27_banana_qa/issue_5765_timezone_support/_01_table_operations/_02_alter/answers/_01_alter_add_hard_default_yes_tsltz.answer_win
```

Reply:
Reflected the review. Please check again.